src/analyses/variable-sensitivity/variable_sensitivity_domain.h

```diff
@@ -78,7 +78,8 @@ class variable_sensitivity_domaint : public ai_domain_baset
   explicit variable_sensitivity_domaint(
     variable_sensitivity_object_factory_ptrt _object_factory,
     const vsd_configt& _configuration)
-    : abstract_state(_object_factory)
+    : abstract_state(_object_factory),
+      flow_sensitivity(_configuration.flow_sensitivity)
```

Comment:
I do wonder if it would be more future-proof to pass the whole `vsd_configt` but this is probably fine for now and it will be quite a small change to add this if needed in the future.